docs/src/main/asciidoc/doc-create-tutorial.adoc

```diff
@@ -29,7 +30,8 @@ For this example, we will use `acme-serve-http-requests-tutorial.adoc`:
 
 - `acme-` will group this guide with other resources related to the acme extension
 - `serve-http-requests` is a derivative of the document title
-- `-tutorial.adoc` will indicate that this document is a tutorial
+- `-tutorial` will indicate that this document is a tutorial (optional)
```

Comment:
```suggestion
```

Reply:
Line 29 and line 3:

If we want to encourage folks to not use the content type in filename, I think the tutorial examples shouldn't mention -tutorial in the adoc name or guidance. WDYT?